lambda/api/endpoint_cloud/api_handler_event.py

```diff
@@ -79,7 +79,13 @@ def create(self, request):
                 try:
                     state = json_object['event']['endpoint']['state']  # Expect a string, ex: powerState
                     state_value = json_object['event']['endpoint']['value']  # Expect string or JSON
-
+                    namespace = json_object['event']['endpoint']['namespace']
```

Comment:
I introduced your changes and then posted the following **ChangeReport** to `https://{{endpoint_api_id}}.execute-api.{{aws_region}}.amazonaws.com/prod/events `using POSTMAN

body: `
{
    "event": {
        "type": "ChangeReport",
        "endpoint": {
            "userId": "{{user_id}}",
            "id": "{{endpoint_id}}",
            "state": "state",
            "value": "OFF",
            "type": "SWITCH",
            "sku": "SW00"
        }
    }
}
`
Looking at the **CloudWatch** logs, I got this error: 
`LOG` api.index.handler.request.api_handler.event.create.response: KeyError: 'namespace'

I noticed that this is because the payload from POSTMAN did not contain a namespace key... Alexa will expect this key and is added when `AlexaResponse.create_context_property(name=state, value=state_value)` is eventually called.

The question is: Should the payload from POSTMAN or the actual event that your IoT thing will send to the API need to include the key value for namespace? My understanding is that it should not really care... Your thoughts?

Reply:
`AlexaResponse.create_context_property()` add a default `Alexa.EndpointHealth` value in case namespace is not specified, which is not correct, so we cannot just remove it. 

Bu I agree with you that this value should not be part of the payload to the `/endpoint` api, and should be fetched instead from DDB (it is the `interface` property in the `capabilities` field in DDB).

Looking closer at the specs [specs](https://developer.amazon.com/docs/device-apis/alexa-interface.html#changereport) the Change Report should also report the state of all interfaces implemented by the endpoint, which this function does not do right now.

The correct solution would be to fetch this data from the ThingShadow, and populate both the `context` and `payload` nodes in the Alexa event. 

I am going to implement the first fixe and add it to the PR. Is it OK?
I will create an issue and a new PR for the second fix once the first is merged.